### ISSUE TITLE
Blink insists that “Spaces are not permitted in the user name.”

### ISSUE THREAD

**Issue by JonathanWheeler** [blinksh/blink#1798]
### Checklist

- [x] Using latest version on the [App Store](http://itunes.apple.com/app/id1156707581)
- [x] Read the [docs](http://blink.sh/docs/)
- [x] Searched for [existing GitHub issues](https://github.com/blinksh/blink/issues)

### Configuration

Blink v16.1.9.766 for iPadOS 16.5.1

### Describe the bug

I can’t connect to one of my hosts using Blink because Blink’s config UI insists with an error that “Spaces are not permitted in the user name.” Spaces in a user name can be annoying, but they’re definitely permitted.

I’ve tried adding this to my SSH config in the config UI to no avail (mirroring what I have on my other systems, but with the user name changed here for security):

`User "Foo Bar"`

I’ve also tried this to no avail:

```bash
blink> cat << 'EOF' > ~/.ssh/config
Host hosta* HostA*
    User "Foo Bar"
^C
```

For now, I just have to manually jump through a different HostB, then `ssh “Foo Bar”@HostA`.

Edit: While using `ssh “Foo Bar”@HostA` in Blink doesn’t work, `ssh Foo\ Bar@HostA` does. I’m now using this.

Thank you for your work.

**Comment by carloscabanero** [comment]
Will try to fix ASAP

**Comment by yury** [comment]
Ok, problem is [here](https://github.com/holzschu/ios_system/blob/6a83eb1c6c383a562fbe33a7e97677e88d305b51/ios_system.m#L2387)
ios_system splits `argv` differently with quotes. For instance, `ssh “Foo Bar”@HostA` would be [`ssh`, `Foo Bar`, `HostA`] instead of [`ssh`, `Foo Bar@HostA`]

**Comment by carloscabanero** [comment]
This is fixed in v16.2.1
